src/pages/projectSettings/tabs/GithubCommitQueueTab/getFormSchema.tsx

```diff
@@ -440,6 +445,20 @@ export const getFormSchema = (
             "ui:hide": true,
           }),
         },
+        requiredApprovalCount: {
+          "ui:data-cy": "required-approval-count-input",
+          ...((formData?.commitQueue?.enabled === false ||
+            !!githubProjectConflicts?.commitQueueIdentifiers?.length) && {
+            "ui:hide": true,
+          }),
```

Comment:
I would double check that this is true, but I believe you can delete this since you handle hiding the field directly below

```suggestion
```

Reply:
done, thanks